mmv1/templates/terraform/custom_flatten/firebase_app_check_debug_token.go.erb

```diff
@@ -0,0 +1,21 @@
+<%# # the license inside this if block pertains to this file
+        # Copyright 2024 Google Inc.
+        # Licensed under the Apache License, Version 2.0 (the "License");
+        # you may not use this file except in compliance with the License.
+        # You may obtain a copy of the License at
+        #
+        #     http://www.apache.org/licenses/LICENSE-2.0
+        #
+        # Unless required by applicable law or agreed to in writing, software
+        # distributed under the License is distributed on an "AS IS" BASIS,
+        # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+        # See the License for the specific language governing permissions and
+        # limitations under the License.
+#%>
+func flatten<%= prefix -%><%= titlelize_property(property) -%>(v interface{}, d tpgresource.TerraformResourceData, config *transport_tpg.Config) interface{} {
+    // `debug_token_id` is the last segment of `name`
+    if err := d.Set("debug_token_id", tpgresource.GetResourceNameFromSelfLink(v.(string))); err != nil {
```

Comment:
If token id is required and ignore_read/url_param_only (which it should be) then we don't need to ever set it in state manually as it will never change from the value we sent.